ui/pages/notifications/notification-components/lido-stake-ready-to-be-withdrawn/lido-stake-ready-to-be-withdrawn.tsx

```diff
@@ -95,6 +96,17 @@ export const components: NotificationComponent<LidoReadyWithDrawnNotification> =
       ),
       body: {
         type: 'body_onchain_notification',
+        Account: ({ notification }) => {
+          if (!notification.address) {
+            return null;
+          }
+          return (
+            <NotificationDetailAddress
+              side={t('account') || ''}
```

Comment:
nit - we don't need to default to empty string. The translation will be there.

Keep as is, it's not a blocker.